website/docs/addon_unreal_artist.md

```diff
@@ -147,47 +178,6 @@ To publish a mesh from Unreal, you need to create a publish instance.
   - On the left, you will see all the publish instances that you have created. 
   - Select the ones that you want to publish, and click on **Publish**.
 
-## Look development
-
-AYON supports look development in Unreal Engine.
-
-### Loading model
-
-First, you will need a model as Static Mesh. To load it: 
-- Click on **AYON → Load ...**.
-- Right-click your mesh.
-- Select **Import Static Mesh**.
-
-![Unreal AYON Tools Load](assets/unreal_ayon_menu_load.png)
-![Unreal AYON Load Model](assets/unreal_load_model.png)
-
-You will find the loaded Static Mesh in `/Content/AYON/Assets`.
-
-### Creating look
-
-To create the look, you will need to create the [Materials](https://docs.unrealengine.com/5.1/en-US/Engine/Rendering/Materials/) for the model you loaded, and assign them to the model. You can create the materials in Unreal, or you can import them from AYON as UAsset.
-
-### Publishing look
-
-To publish a look from Unreal, you need to create a publish instance.
-
-- Select the model with the materials in the Content Browser.
-- Click on **AYON → Publisher ...** to open the Publisher screen.
-  - On the top bar, switch to the **Create** tab.
-  - In the Creator screen, select *Look*.
-  - Set the name of the product.
-  - Click on **Create**. This will create a `AyonPublishInstance` file in `/Content/AYON/PublishInstances`, with the metadata necessary to publish the asset.
-
-![Unreal AYON Tools Publisher](assets/unreal_ayon_menu_publisher.png)
-![Unreal AYON Create Look](assets/unreal_create_look.png)
-
-- Click on **AYON → Publisher ...** to open the Publisher screen.
-  - On the top bar, switch to the **Publish** tab.
-  - On the left, you will see all the publish instances that you have created. 
-  - Select the ones that you want to publish, and click on **Publish**.
-
-To get the look ready to be publishable, AYON will create a new folder `/Content/AYON/Looks`. In here, it will be created a folder that will contain a simple model with the materials you created assigned, that will be used to publish the look. This folder will be named after product name you chose when creating the instance. You don't need to do anything with this folder, it is just for the publishing process.
-
```

Comment:
Is this feature/fixing this feature is in the backlog?
if yes, we can remove it temporarily till we fix it.
```suggestion
<!--  
## Look development

AYON supports look development in Unreal Engine.

### Loading model

First, you will need a model as Static Mesh. To load it: 
- Click on **AYON → Load ...**.
- Right-click your mesh.
- Select **Import Static Mesh**.

![Unreal AYON Tools Load](assets/unreal_ayon_menu_load.png)
![Unreal AYON Load Model](assets/unreal_load_model.png)

You will find the loaded Static Mesh in `/Content/AYON/Assets`.

### Creating look

To create the look, you will need to create the [Materials](https://docs.unrealengine.com/5.1/en-US/Engine/Rendering/Materials/) for the model you loaded, and assign them to the model. You can create the materials in Unreal, or you can import them from AYON as UAsset.

### Publishing look

To publish a look from Unreal, you need to create a publish instance.

- Select the model with the materials in the Content Browser.
- Click on **AYON → Publisher ...** to open the Publisher screen.
  - On the top bar, switch to the **Create** tab.
  - In the Creator screen, select *Look*.
  - Set the name of the product.
  - Click on **Create**. This will create a `AyonPublishInstance` file in `/Content/AYON/PublishInstances`, with the metadata necessary to publish the asset.

![Unreal AYON Tools Publisher](assets/unreal_ayon_menu_publisher.png)
![Unreal AYON Create Look](assets/unreal_create_look.png)

- Click on **AYON → Publisher ...** to open the Publisher screen.
  - On the top bar, switch to the **Publish** tab.
  - On the left, you will see all the publish instances that you have created. 
  - Select the ones that you want to publish, and click on **Publish**.

To get the look ready to be publishable, AYON will create a new folder `/Content/AYON/Looks`. In here, it will be created a folder that will contain a simple model with the materials you created assigned, that will be used to publish the look. This folder will be named after product name you chose when creating the instance. You don't need to do anything with this folder, it is just for the publishing process.
-->
```

Reply:
> @moonyuet Is this feature/fixing this feature is in the backlog? if yes, we can remove it temporarily till we fix it.

I had the catchup call with @antirotor and @LiborBatek last Friday. As this look publish is broken and we dont have the use case for that, we can just remove this excerpt to cause less confusion of the users when they are diving into the docs for answers. I have already created PR to remove the related creator too.